src/pipeline.rs

```diff
@@ -135,6 +134,7 @@ impl<Depth> Pipeline<Depth> {
 
         if instances.len() > self.supported_instances {
             self.instances = device.create_buffer(&wgpu::BufferDescriptor {
+                label: Some("instances"),
```

Comment:
It would probably be better to use "wgpu_glyph.instances" or something like that so there is some context if it shows up in a larger trace

Reply:
Agreed :+1: